scripts/fetch_places_and_screens.exs

```diff
@@ -272,6 +272,14 @@ live_screens =
         %{id: id, type: app_id, disabled: disabled, direction_id: direction_id}
     end
   )
+  |> Enum.reduce(%{}, fn {id, screen_list}, acc ->
```

Comment:
I think this can just be a `Map.new` or `Enum.into` since the number of pairs in the map doesn't change. You're just mapping the value of each pair to a new one, so the mapper would look like
```ex
fn {id, screen_list} ->
  all_but_one_triptych = ...
  {id, all_but_one_triptych}
end
```